examples/flow-counter/interfaces/redux-thunk.js

```diff
@@ -0,0 +1,13 @@
+import type { Middleware, MiddlewareAPI, Dispatch, Store } from 'redux';
+
+declare module 'redux-thunk' {
+  declare type Thunk<State, Dispatchable> = (dispatch: Dispatch<Dispatchable>, getState: () => State) => ?Dispatchable;
+  declare type ThunkMiddlewareDispatchable<State, Dispatchable> = Thunk<State, Dispatchable> | Dispatchable;
+
+  // Cannot find a way to export the default function with the correct type signature
+  // declare var exports: {
```

Comment:
I couldn't find a way make the default module export be a polymorphic function in Flow. Does someone know how to do this? If not, probably is a bug or something yet unsupported in Flow.